yolo_to_yolo/scripts.py

```diff
@@ -0,0 +1,61 @@
+import os
+import shutil
+import re
+from tkinter import Tk, filedialog
+
+# Function to open a folder dialog
+def select_directory(title):
```

Comment:
Please add type annotations, especially for function parameters. Python lets us slack off, but that doesn't mean that we should.

Also, this function should be called `prompt_for_directory`, and the parameter should be called `prompt` or `prompt_text`.